src/web/assets/cp/src/js/BaseElementIndex.js

```diff
@@ -1990,12 +1990,12 @@ Craft.BaseElementIndex = Garnish.Base.extend(
         this.updateFilterBtn();
       }
 
+      this.onSelectSource();
+
```

Comment:
Fire `onSelectSource` before `onSourcePathChange`, so our inputs and buttons are created first.